client/src/href-builder.js

```diff
@@ -52,7 +53,11 @@ module.exports = function(config){
 
     prepareServerGet: function(baseUrl, component, options) {
       var urlPath = component.name + (component.version ? '/' + component.version : '');
-      var qs = options.parameters ? ('/?' + querystring.stringify(options.parameters)) : '';
+
+      var qs = '';
+      if (component.parameters || options.parameters) {
+        qs = '/?' + querystring.stringify(mergeObjects(component.parameters, options.parameters));
```

Comment:
I was thinking about that but looking at the package.json I couldn't see `lodash` as a dependency (https://github.com/opentable/oc/blob/master/client/package.json#L17-L21) also in ES6 this will be part of the language so I preferred not to add a dependency and go "vanilla". plus the component is a standalone one (and fully tested) so we can always swap its internal implementation.